src/util.js

```diff
@@ -0,0 +1,37 @@
+export const getAllMovies = () => {
+return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')  
+  .then(response => { 
+      console.log(response.status)                  
+      if(!response.ok) {
+        throw new Error('Something went wrong.');
+      } else if (response.status === 500) {
```

Comment:
yeah, what if it's a 404? maybe we could have a conditional check if it's in the 400 range, or if it's great that 499, do this...